pkg/config/controllerconfig.go

```diff
@@ -52,6 +52,9 @@ type ControllerConfig struct {
 
 	// EnableMetricsTransport enables automatic wrapping of HTTP clients with metrics transport
 	EnableMetricsTransport bool
+
+	// Use the legacy/ handwritten IAM controllers.
+	UseLegacyIAM bool
```

Comment:
The default value is false which means we use the new IAM controller directly. Another option is to use `UseK8sIAM` (like opt-in), , the legacy would be the default and then we can flip the bit to use k8s in the next 1-2 versions.

Reply:
I think per my chatter with @cheftako we might want to turn this on by default and not require practitioners/ CXs to opt in.